src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/ReadFromCsvWriteToExcel.java

```diff
@@ -0,0 +1,64 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.core.Record;
+import com.northconcepts.datapipeline.foundations.file.LocalFile;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.action.convert.ConvertStringToNumberCommand;
+import com.northconcepts.datapipeline.foundations.pipeline.action.transform.AddFieldsCommand;
+import com.northconcepts.datapipeline.foundations.pipeline.action.transform.RenameFieldsCommand;
+import com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput;
+import com.northconcepts.datapipeline.foundations.pipeline.output.ExcelPipelineOutput;
+import com.northconcepts.datapipeline.job.Job;
+
+import java.nio.channels.Pipe;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ReadFromCsvWriteToExcel {
+
+    public static void main(String[] args) throws Throwable{
+
+        Pipeline pipeline = new Pipeline();
+
+        LocalFile inputFile = new LocalFile()
+                .setName("Input File")
+                .setPath("PATH_TO_INPUT_FILE.csv")
+                .detectFileType();
+
+        CsvPipelineInput pipelineInput = new CsvPipelineInput()
+                .setFileSource(inputFile)
+                .setFieldNamesInFirstRow(true);
+
+        LocalFile outputFile = new LocalFile()
+                .setName("Output File")
+                .setPath("PATH_TO_OUTPUT_FILE.xlsx")
```

Comment:
Let's make sure to use a real file to make this a working example.

Reply:
done